third_party/move/move-prover/bytecode-pipeline/src/packed_types_analysis.rs

```diff
@@ -74,27 +75,14 @@ pub fn get_packed_types(
     packed_types
 }
 
-#[derive(Debug, Clone, Default, Eq, PartialOrd, PartialEq)]
+#[derive(AbstractDomain, Debug, Clone, Default, Eq, PartialOrd, PartialEq)]
 struct PackedTypesState {
```

Comment:
Don't you need to add #[join] for all the fields here?

Reply:
done